code/go/0chain.net/smartcontract/dbs/event/config.go

```diff
@@ -0,0 +1,258 @@
+package event
+
+import (
+	"time"
+
+	"0chain.net/chaincore/config"
+	"0chain.net/core/datastore"
+	"github.com/0chain/common/core/currency"
+)
+
+type TestConfig struct {
```

Comment:
Test structs need to be put in `_test.go` file.

Reply:
it's not test struct, it is utility struct for testing